### PR TITLE
Remove CoffeeScript dependencies and use vanilla JavaScript

### PR DESCRIPTION
**Pull request by DimaKaUA**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,14 +10,15 @@ gem 'sass-rails', '~> 5.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.1.0'
+# gem 'coffee-rails', '~> 4.1.0'
```
Comment (rakvium) [inline Gemfile]: Don't comment, just remove.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -2,9 +2,7 @@
 <html>
 <head>
   <title>Taxiproject</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (rakvium) [inline app/views/layouts/application.html.erb:6]: Why did you remove these lines?

**Review by rakvium — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (DimaKaUA) [reply Gemfile]: will be done

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (DimaKaUA) [reply app/views/layouts/application.html.erb:6]: if i understood correctly this code relates to coffeescript

Comment (DimaKaUA) [reply app/views/layouts/application.html.erb:6]: and it is not suitable now

Comment (DimaKaUA) [reply app/views/layouts/application.html.erb:6]: This caused an error)))

Comment (rakvium) [reply app/views/layouts/application.html.erb:6]: @DimaKaUA, no, it's just JavaScript.

Comment (DimaKaUA) [reply app/views/layouts/application.html.erb:6]: But it doesn't work after removal of gem "coffee-rails". I have not figured out with this

File: Gemfile [hunk 3]
```diff
@@ -10,14 +10,15 @@ gem 'sass-rails', '~> 5.0'
...
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 
 # Our gems
 gem 'devise'
 gem 'cancancan'
 gem 'rubocop'
+gem 'vanilla-ujs'
```
Comment (rakvium) [inline Gemfile]: Are you serious? Remove this gem, just don't use any CoffeeScript.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (rakvium) [reply app/views/layouts/application.html.erb:6]: Search for `coffee` in config files. Remove any inclusion of it.

**Review by rakvium — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
+ // This is a manifest file that'll be compiled into application.js, which will include all the files
 // listed below.
 //
 // Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
```
Comment (rakvium) [inline app/assets/javascripts/application.js]: Remove the `Coffee`, please :wink:

File: db/schema.rb [hunk 5]
```diff
@@ -11,30 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161003144644) do
+ActiveRecord::Schema.define(version: 20161003140410) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "dispatchers", force: :cascade do |t|
```
Comment (rakvium) [inline db/schema.rb:18]: Please pull current master branch. We don't need to remove dispatchers :wink:

**Review by rakvium — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/orders.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (rakvium) [inline app/assets/javascripts/orders.js]: Please use JS comments :wink:

**Review by rakvium — CHANGES_REQUESTED** [review]

File: app/controllers/orders_controller.rb [hunk 7]
```diff
@@ -1,6 +1,6 @@
 class OrdersController < ApplicationController
   before_action :set_order, only: [:show, :edit, :update, :destroy]
-
+
```
Comment (rakvium) [inline app/controllers/orders_controller.rb]: Redundant space

**Review by rakvium — CHANGES_REQUESTED** [review]

**Review by rakvium — APPROVED** [review]

**Pull request merged**